backend/app/environments/schema.py

```diff
@@ -18,27 +17,43 @@ class GetEnvironment(Environment):
 
 
 class _AWSS3Config(ORMModel):
-    account_id: int
-    name: str
+    identifier: str
+    bucket_name: str
     arn: str
-    kms: str
+    kms_key: str
+    is_default: bool
 
 
-class Config(RootModel[Dict[str, _AWSS3Config]]):
-    pass
+class _AWSGlueConfig(ORMModel):
```

Comment:
I would put all of these configs in separate files

Reply:
similar to my approach in data outputs branch

Reply:
How do you suggest splitting it? Based on which entity?

Reply:
Have a file per service. I have S3DataOutput and GlueDataOutput separated, but inheriting from a generic DataOutput.
Same would go for these configs.

Reply:
Fixed